taichi/python/export_lang.cpp

```diff
@@ -777,10 +778,11 @@ void export_lang(py::module &m) {
 
   m.def("make_rand_expr", Expr::make<RandExpression, const DataType &>);
 
-  m.def("make_const_expr_i32", Expr::make<ConstExpression, int32>);
-  m.def("make_const_expr_i64", Expr::make<ConstExpression, int64>);
-  m.def("make_const_expr_f32", Expr::make<ConstExpression, float32>);
-  m.def("make_const_expr_f64", Expr::make<ConstExpression, float64>);
+  m.def("make_const_expr_ip",
```

Comment:
What does `ip` stand for? IIUC `fp` is the acronym of floating point, and the acronym of integer should be `int`?

Reply:
good point, I don't know what ip stands for either but copied from default_ip lol. Changed to int here ;)